Sources/Build/BuildDescription/SwiftTargetBuildDescription.swift

```diff
@@ -392,6 +392,18 @@ public final class SwiftTargetBuildDescription {
         }
         #endif
 
+        // If we're using an OSS toolchain, add the required arguments bringing in the plugin server from the default toolchain if available.
+        if self.buildParameters.toolchain.isSwiftDevelopmentToolchain, driverSupport.checkSupportedFrontendFlags(flags: ["-external-plugin-path"], toolchain: self.buildParameters.toolchain, fileSystem: self.fileSystem), let pluginServer = self.buildParameters.toolchain.swiftPluginServerPath {
```

Comment:
not for this PR but we should change `checkSupportedFrontendFlag` to take an enum instead of a string so we can centralize the usage of the string expression

Reply:
You mean for the flags we're checking? Seems a bit odd to do that specifically for `checkSupportedFrontendFlag` when we're using literal strings all over the `Build` module. I could see a broader change that uses a typed way for this everywhere, but IMO should be coming from the swift-driver library.

Reply:
yea, the problem is that all these strings everywhere can get out of sync and are focused on actual CLI invocation instead of on the capabilities. I think the call sites would be better if they look more like the following:

```
if driverSupport.supports(.externalPlugin) {
    ...
    args += driverSupport.flag(for: .externalPlugin, with: value)
}
```